packages/core/test/scuttle.spec.js

```diff
@@ -0,0 +1,126 @@
+const test = require('ava')
+const { evaluateWithSourceUrl } = require('./util')
+const { scuttle } = require('../src/scuttle')
+
+const SCUTTLER_NAME_ERROR = {
+  message:
+    'LavaMoat - \'scuttlerName\' function "SCUTTLER" expected on globalRef.' +
+    'To learn more visit https://github.com/LavaMoat/LavaMoat/pull/462.',
+}
+
+const err = (intrinsic) =>
```

Comment:
(style) guess I would go for templates here instead of string concatenation

Reply:
I want lines of code to remain short, this practice is similar in the source code too

Reply:
something to be said for consistency, I suppose!

another style issue would be use a template on L7 above such that we do not need to escape anything

Reply:
length of lines should be a concern for machines, not humans ;) 
If linter permits it, use a template string.
On the other hand, if linter permits it, use whatever string concatenation you want.